docs/GoGetSllDoc.md

```diff
@@ -0,0 +1,669 @@
+            "record": ""
+        }
+    },
+    "order_id": "",
+    "order_status": "",
+    "success": ""
+}</pre>
+
+<h3>operation: Add SSL Renew Order</h3>
+
+<h4>Input parameters</h4>
+
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>Product ID</td><td>Specify the product ID for which the SSL certificate is being renewed.</td></tr>
+<tr><td>Certificate Signing Request (CSR)</td><td>Specify the CSR (Certificate Signing Request) encoded text for the certificate you want to renew.</td></tr>
+<tr><td>Server Count</td><td>Specify the number of servers or instances that will use the SSL certificate being renewed.</td></tr>
+<tr><td>Period</td><td>Specifies the validity duration of the renewed SSL certificate.</td></tr>
+<tr><td>Approver Emails</td><td>Specifies the email addresses of the individuals who are authorized to approve the renew of the SSL certificate.</td></tr>
+<tr><td>Additional Field</td><td>(Optional) Specify any additional fields required for the use case, and provide them in JSON format.</td></tr>
+</tbody></table>
+
+<h4>Output</h4>
+
+<p>The output contains the following populated JSON schema:</p>
+
+<pre>{
+    "product_id": "",
+    "approver_method": {
+        "http": {
+            "link": "",
+            "filename": "",
+            "content": ""
+        }
+    },
+    "order_id": "",
+    "invoice_id": "",
+    "order_status": "",
+    "success": "",
+    "san": [
+        {
+            "san_name": "",
+            "validation_method": "",
+            "status": 1,
+            "status_description": "",
+            "validation": {
+                "http": {
+                    "link": "",
+                    "filename": "",
+                    "content": ""
+                }
+            }
+        }
+    ],
+    "order_amount": "",
+    "currency": "",
+    "tax": "",
+    "tax_rate": ""
+}</pre>
+
+<h3>operation: Add SSL SAN Order</h3>
+
+<h4>Input parameters</h4>
+
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>Order ID</td><td>Specify the order ID for which you want to add an SSL certificate with Subject Alternative Names (SANs).</td></tr>
+<tr><td>Wildcard SAN Count</td><td>(Optional) Specify the number of wildcard SANs (e.g., *.example.com) to be included in the certificate.</td></tr>
+<tr><td>Single SAN Count</td><td>(Optional) Specify the number of single SANs (e.g., www.example.com) to be included in the certificate.</td></tr>
+</tbody></table>
+
+<h4>Output</h4>
+
+<p>The output contains a non-dictionary value.</p>
+
+<h3>operation: Cancel Order</h3>
+
+<h4>Input parameters</h4>
+
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>Order ID</td><td>Specify the unique identifier of the SSL certificate order that you want to cancel.</td></tr>
+<tr><td>Reason</td><td>(Optional) Specify the reason for why the SSL certificate order is being canceled.</td></tr>
```

Comment:
Update the description:

e.g.Specify the reason why the SSL certificate order is being canceled.